### PR TITLE
Added example with-now-v2

### PR DESCRIPTION
**Pull request by dfreire**
Straightforward example to deploy to now v2.

Also see #900

### CODE REVIEW

File: examples/with-now-v2/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "razzle-examples-now-v2",
+  "version": "2.4.1",
+  "license": "MIT",
+  "scripts": {
+    "start": "razzle start",
+    "build": "razzle build",
+    "test": "razzle test --env=jsdom",
+    "start:prod": "NODE_ENV=production node build/server.js"
```
Comment (sync) [inline examples/with-now-v2/package.json]: You could also add a now-build script

File: examples/with-now-v2/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  "dependencies": {
+    "express": "^4.16.4",
+    "razzle": "^2.4.1",
```
Comment (jaredpalmer) [inline examples/with-now-v2/package.json]: Duplicate deps. Only include devDep

File: examples/with-now-v2/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "react": "^16.8.3",
+    "react-dom": "^16.8.3",
+    "react-router-dom": "^4.3.1"
```
Comment (jaredpalmer) [inline examples/with-now-v2/package.json]: Remove react router since it isn’t used

**Review by jaredpalmer — CHANGES_REQUESTED** [review]
Few nits

**Comment by gl0gl0** [comment]
I've tried this, but I'm getting an error after now deployment that says:
`INTERNAL_MISSING_RESPONSE_FROM_CACHE`
Any ideas? I can't find any documentation about it.

**Pull request merged**